imports/collections/schemas/accounts.js

```diff
@@ -187,7 +187,8 @@ export const Accounts = new SimpleSchema({
   "groups": {
     type: Array, // groupIds that user belongs to
     optional: true,
-    defaultValue: []
+    defaultValue: [],
+    index: 1
```

Comment:
I don't know what our current status is on adding indexes: https://github.com/reactioncommerce/reaction/issues/4407
But if you need this index for the script to perform efficiently we should add it.

My guess is that there aren't that many groups that it would need a compulsory index.

Reply:
@Akarshit While there aren't a lot of groups, there are a lot of accounts, which is I added this index to Accounts.groups. If I don't have this index, wouldn't Mongo have to scan all 2.1 million accounts? I'm running the retailer dataset